fenix/app/src/main/res/layout/pbm_shortcut_popup.xml

```diff
@@ -42,7 +42,7 @@
             android:id="@+id/cfr_pos_button"
             style="@style/NeutralButton"
             android:layout_width="0dp"
-            android:layout_height="36dp"
+            android:layout_height="wrap_content"
```

Comment:
This would lead to a significant change in UI:

| before | after |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/11428869/226296635-6f28d7eb-4151-409b-945e-271c728d064a.png) | ![image](https://user-images.githubusercontent.com/11428869/226296730-e99635e3-2fc7-4010-aa75-b8a728942e76.png) |

Easiest way to keep the current UI would be to add 6dp vertical insets to the background similar to [what we're doing now to increase tap area](https://github.com/mozilla-mobile/firefox-android/blob/795f3950b671b102de99b47684a7089ffef10b96/fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt#L910) (line of code which after this changes should be removed).

Could also check with UX for an approval for visually increasing the button height.
@gitstart Let me know what you prefer.

Reply:
@Mugurell can i look into it and correct the bug without changing much into the ui ?

Reply:
okay, I'll look into it:).